rps.js

```diff
@@ -0,0 +1,39 @@
+const playGame = function (userChoice){
+console.log("your choice is : " + userChoice);
+    let CompuChoose = Math.floor(Math.random() * 3);
+    if (CompuChoose === 0) {
+        CompuChoose = 'rock';}
+    if (CompuChoose === 1) {
+        CompuChoose = 'Paper';}
+    if (CompuChoose === 2) {
+        CompuChoose = 'Scissor';}
+        console.log("computer choice :" + CompuChoose );
+    // if (rock !== CompuChoose){
+    //     console.log("win");}else{
+    //     console.log("lose");} 
+    if (userChoice === CompuChoose) { console.log("It's a tie!"); }
```

Comment:
using if else in your if statements will make this code function better, currently your code will not print the final results

its better to write it like this.

` if (userChoice === CompuChoose) { console.log("It's a tie!"); }
   else  if (userChoice ==='rock'){
        if(CompuChoose ==='paper'){
            console.log("you lose !!");
        }else{
            console.log("you win");
        }
    }
    else if (userChoice === 'paper') {
        if (CompuChoose === 'Scissor') {
            console.log("you lose !!");
        } else {
            console.log("you win");
        }
    }
    else if (userChoice === 'Scissor') {
        if (CompuChoose === 'rock') {
            console.log("you lose !!");
        } else {
            console.log("you win");
        }
    }

}`